clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/images/repo/FileResourcesRepoFactory.kt

```diff
@@ -0,0 +1,46 @@
+package com.clevertap.android.sdk.inapp.images.repo
+
+import android.content.Context
+import com.clevertap.android.sdk.Logger
+import com.clevertap.android.sdk.inapp.images.FileResourceProvider
+import com.clevertap.android.sdk.inapp.images.cleanup.FileCleanupStrategy
+import com.clevertap.android.sdk.inapp.images.cleanup.FileCleanupStrategyExecutors
+import com.clevertap.android.sdk.inapp.images.preload.FilePreloaderExecutors
+import com.clevertap.android.sdk.inapp.images.preload.FilePreloaderStrategy
+import com.clevertap.android.sdk.inapp.store.preference.StoreRegistry
+
+internal class FileResourcesRepoFactory {
```

Comment:
this can be an object

Reply:
Same as https://github.com/CleverTap/clevertap-android-sdk/pull/619#discussion_r1676035340